packages/react/src/components/Icon/Icon.tsx

```diff
@@ -93,7 +93,7 @@ class Icon extends UIComponent<WithAsProp<IconProps>, any> {
   }
 }
 
-Icon.create = createShorthandFactory({ Component: Icon, mappedProp: 'name' })
+Icon.create = createShorthandFactory({ Component: Icon, mappedProp: 'name', primitive: true })
```

Comment:
`primitive` is not very clear to me.  What about `doesSupportJSX` or `allowJSX`, something explicit?

Reply:
Replaced with `allowsJSX`